pkg/sfu/receiver.go

```diff
@@ -242,7 +243,8 @@ func (w *WebRTCReceiver) AddUpTrack(track *webrtc.TrackRemote, buff *buffer.Buff
 		return
 	}
 
-	buff.SetLogger(w.logger)
+	layer := RidToLayer(track.RID())
+	buff.SetLogger(logger.Logger(logr.Logger(w.logger).WithValues("layer", layer)))
```

Comment:
setting logger of `Buffer` with proper layer. It was becoming super hard to debug layer specific stuff without it by looking up SSRC.